test/test-servers/koa_scaffold_server.ts

```diff
@@ -31,18 +32,18 @@ app.use(function*(this, next) {
 });
 
 app.use(function*(this, next) {
-  var start = new Date() as any;
+  const start = new Date() as any;
   yield next;
-  var ms = new Date() as any - start;
+  const ms = new Date() as any - start;
   this.set('X-Response-Time', ms + 'ms');
 });
 
 // logger
 
 app.use(function*(this, next) {
-  var start = new Date() as any;
+  const start = new Date() as any;
```

Comment:
This doesn't feel right.  Why are you casting the Date to any?

Reply:
Yes, we should be using `Date.now()`.  I've added a TODO since I think this should be addressed in a follow-up PR.

Reply:
Issues are better than TODO comments.

Reply:
I'll create an issue.